packages/common/src/hooks/purchaseContent/validation.ts

```diff
@@ -35,6 +39,11 @@ const createPurchaseContentSchema = () => {
       [GUEST_EMAIL]: z.string().email().optional(),
       [PURCHASE_METHOD_MINT_ADDRESS]: z.string().optional()
     })
+    .merge(
```

Comment:
do we need to merge? why not just have it in the object?

Reply:
just trying to re-use the same logic as sign up. but it does have issues if you try to purchase again you have an email that exists but in this case it's ok. so re-using might not be best